xgcm/grid.py

```diff
@@ -2346,29 +2375,6 @@ def transform(self, da, axis, target, **kwargs):
         ax = self.axes[axis]
         return ax.transform(da, target, **kwargs)
 
-    @docstrings.dedent
```

Comment:
I moved this up, because it was kind of weird to have those methods split far apart in the code.